README.md

```diff
@@ -62,13 +60,26 @@ updates to your project whenever a new template version is released.
 To create the project, run the following command and follow the prompts:
 
 ```bash
-cruft create https://github.com/scverse/cookiecutter-scverse
+# uv will download all dependencies on-the-fly
+uv tool run --with pre-commit cruft create https://github.com/scverse/cookiecutter-scverse
 ```
 
 This will create a git repository generated from the template.
 Now `cd` into the newly created directory and make the initial commit!
 Don't forget to create a repository on GitHub and upload your project.
 
+<details>
+<summary>Working without `uv`</summary>
+If you prefer to manage the dependencies yourself, you can run
+
+```bash
+pip install cruft pre-commit
```

Comment:
maybe rephrase as “if you want to permanently install the tools”

and tell them to use `uv tool install` or `pipx install`?

Reply:
I was more thinking about someone who doesn't want to or cannot use any additional tools... but since `uv` itself can be installed with pip it's not a good argument. 

So I'm fine with that